apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -106,7 +106,7 @@ export function CommandMenu({ open, onOpenChange }: CommandMenuProps) {
 
     return searchDocumentsData.map((document) => ({
       label: document.title,
-      path: document.path,
+      path: document.teamId ? `/t/${document?.team?.url}/documents/${document.id}` : document.path,
```

Comment:
I feel like this would cause some issues with this statement which produces the above `document.path`

```
path: isOwner(document, user) ? `/documents/${document.id}` : getSigningLink(Recipient, user)
```

Can you move this logic into the `search-documents-with-keyword` instead? And maybe use the `${formatDocumentsPath(team?.url)}/${document.id}` logic instead